data/data/openstack/masters/sg-worker.tf

```diff
@@ -194,6 +313,18 @@ resource "openstack_networking_secgroup_rule_v2" "worker_ingress_services_tcp" {
   description       = local.description
 }
 
+resource "openstack_networking_secgroup_rule_v2" "worker_ingress_services_tcp_v6" {
+  count             = length(var.machine_v6_cidrs)
+  direction         = "ingress"
+  ethertype         = "IPv6"
+  protocol          = "tcp"
+  port_range_min    = 30000
+  port_range_max    = 32767
+  remote_ip_prefix  = element(var.machine_v6_cidrs, count.index)
+  security_group_id = openstack_networking_secgroup_v2.worker.id
+  description       = local.description
+}
```

Comment:
This feels like something that should have been here since dual stack support, i.e. IPv6 LoadBalancer Services need this.

Reply:
I thought about this, but when I tried connecting to a IPv6 LB without it in tech preview, it worked fine.